### PR TITLE
Updated lint config

### PR DESCRIPTION
**Pull request by gerardpaapu**
- upgrade eslint to 8.08
- upgrade eslint-config-eda to 0.3.5
- add prettier at 2.5.1
- add prettier config

this is part of: https://github.com/dev-academy-programme/teaching-tasks/issues/17

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-      .then((greeting) => {
-        console.log(greeting)
-        setGreeting(greeting)
-      })
+    /* eslint-disable promise/catch-or-return, promise/always-return */
+    getGreeting().then((greeting) => {
+      console.log(greeting)
+      setGreeting(greeting)
+    })
+    /* eslint-enable */
```
Comment (EliseWei) [inline client/components/App.jsx:14]: Can you say a bit about making an exception for this rule, here?

**Review by EliseWei — COMMENTED** [review]
One question. Otherwise good.

File: client/components/App.jsx (hunk 1 above)
Comment (gerardpaapu) [reply client/components/App.jsx:14]: This one is a little too involved to fix in this task, I'll open an issue on teaching-tasks to tidy this up.

**Review by EliseWei — APPROVED** [review]
Looks good!
thank-you1

File: client/components/App.jsx (hunk 1 above)
Comment (gerardpaapu) [reply client/components/App.jsx:14]: https://github.com/dev-academy-programme/teaching-tasks/issues/20 I've created this

**Pull request merged**